docs/development/environment.md

```diff
@@ -2,11 +2,14 @@
 
 ## Development
```

Comment:
@mickmister should any of this go into CONTRIBUTING.md?  then we refer to that?

Reply:
@aaronrothschild Yes, I looked for a `CONTRIBUTING.md` file but it doesn't exist. Should that be done in this PR?

Reply:
I'm not sure useful it would be considering we already have the development docs elsewhere. I think having dev docs in three places (CONTRIBUTING.md would just be proxying to the gitbook docs most likely) is a bit much. 0/5

Reply:
We have "Development" in most of the plugin repos - and @hanzei wrote a great summary which I'm going to use across the repos as it links to the server/web app pages as the single source of truth (vs having individual instructions on each ``README.md). 

``CONTRIBUTING.md`` appears in all our other repos though so one option is to rename "Development" to "Contributing" so even if there isn't a ``CONTRIBUTING.md`` file, there's still consistency across the docs in that respect.

Reply:
For e.g. mattermost-server there is a `CONTRIBUTING.md`, which also just links to our docs.

I'm 2/5 that a `README.md` is the first place that someone who is interested in contributing looking first and not `CONTRIBUTING.md`. Given that I don't think it's worth to have a separate  `CONTRIBUTING.md`.

Reply:
@aaronrothschild Thoughts on the above? Also what's the process of getting this onto the gitbooks branch?

Reply:
@hanzei @justinegeffen Do you think this is good to merge based on discussion above?

Reply:
I'm fine with merging it as it is.